### PR TITLE
For the Station's own good [SPEEDMERGE]

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

to screw over the mime. was requested.

so if the mime breaks their vow, they get BSA'd and lose all of their blood. so don''t break it.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
5000 bloodloss :^^^^^^^)

:+1: speedmerge

**Comment by oisin100** [comment]
How long till the non coders realize.

File: code/datums/spells/mime.dm [hunk 1]
```diff
@@ -59,4 +59,15 @@
...
\ No newline at end of file
+			H << "<span class='notice'>You break your vow of silence.</span>"
+/*
+			sleep(50)
+			H << "<span class='notice'>Centcomm noticed.</span>
+			explosion(H.loc, 0, 0, 0, 0)
+			adjustBruteLoss(min(99,(H.health - 1)))
+			H.Stun(20)
+			H.Weaken(20)
+			var/area/cornered = get_area(H)
```
Comment (oisin100) [inline code/datums/spells/mime.dm:70]: You should probably recalibrate the antagonist mainframe here to make sure you dont get any memory leaks and stop the LynchExceptionErrors

**Comment by ghost** [comment]
yes

**Comment by AsV9** [comment]
I don't understand this meme.

**Comment by ShadowDeath6** [comment]
Meme is that shitty mimes become second clowns and if they talk they should get saladed for it. 
Mime job is a commitment, not something you can throw around lightly.

**Comment by DavidEdwards** [comment]
I would actually be open to CentCom related punishment for breaking mime silence. Come up with something a little more serious and we can actually go forward with it :D

**Pull request closed without merging**